simpler_eda/corr_map.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+import altair as alt
+import numpy as np
+
+
+def corr_map(data, features, color_schme='yellowgreenblue', plot_width=400, plot_height=300, title="Correlation Map"):
+  """
+  Plot a correlation map with the given dataframe object and a list of numerical features. 
+  Users are allow to set multiple arguments regarding the setting of the correlation plot 
```

Comment:
Users are **_allowed_** to set multiple arguments regarding the setting of the correlation plot **_including_** color schemes, plot width, height and plot title.